src/iam.ts

```diff
@@ -1498,7 +1503,7 @@ export class IAM extends IAMBase {
         subject,
         registrationTypes = [RegistrationTypes.OffChain],
     }: {
-        claim: { claimType: string; claimTypeVersion: number; fields: { key: string; value: string | number }[] };
+        claim: { claimType: string; claimTypeVersion: number };
```

Comment:
I think in `createClaimRequest` we want to include the `fields` so that the issuer can read them. It is only at the time of issuing that we want to remove the `fields`